lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -181,26 +181,25 @@ def test_start_flow_not_verified(self, course_mode, payment_flow):
     @httpretty.activate
     @override_settings(
         ECOMMERCE_API_URL=TEST_API_URL,
-        ECOMMERCE_PUBLIC_URL_ROOT=TEST_PUBLIC_URL_ROOT
```

Comment:
TEST_PUBLIC_URL_ROOT looks unused now. Could you delete it from imports and maybe from the source file (edx-platform/lms/djangoapps/commerce/tests/__init__.py) too

Reply:
updated